### ISSUE TITLE
About BeginTabBar/EndTabBar：Can't display normally

### ISSUE THREAD

**Issue by YuDang1024** [ocornut/imgui#3331]
I am using the TabBar function in Dear ImGui
I want to make the same example in Demo：
![image](https://user-images.githubusercontent.com/55690445/86330308-13a78880-bc7a-11ea-81ba-7362cbadbfe2.png)
But it seems not very successful：
![image](https://user-images.githubusercontent.com/55690445/86330663-96c8de80-bc7a-11ea-8360-f397d544c462.png)
I don't know where is the problem, or is there a wrong way of using it?
this is my code ，Looks like no problem：
`    _, self.m_bOpen = ImGui.Begin("FaceAbout", self.m_bOpen);
    ImGui.Separator()
    if ImGui.BeginTabBar("#tabs",ImGuiTabBarFlags_.ImGuiTabBarFlags_None) then
        if ImGui.BeginTabItem("FaceBone") then
            ImGui.Text("Hello")
            ImGui.EndTabItem()
        end
        if ImGui.BeginTabItem("FaceDecal") then
            ImGui.Text("Say")
            ImGui.EndTabItem()
        end
        if ImGui.BeginTabItem("FaceFile") then
            ImGui.Text("Babay")
            ImGui.EndTabItem()
        end

        ImGui.EndTabBar()
    end
    ImGui.End();`

**Comment by YuDang1024** [comment]
I am using Lua script
But maybe this is not an important reason

**Comment by sonoro1234** [comment]
> I am using Lua script
> But maybe this is not an important reason

Or may be it is an important reason: Which Lua library are you using, version, system ...

**Comment by sonoro1234** [comment]
With https://github.com/sonoro1234/LuaJIT-ImGui
```Lua
local igwin = require"imgui.window"
local win = igwin:GLFW(800,400, "tabissue")

function win:draw(ImGui)
	ImGui.Begin("FaceAbout", self.m_bOpen);
	ImGui.Separator()
	if ImGui.BeginTabBar("#tabs", ImGui.lib.ImGuiTabBarFlags_None) then
		if ImGui.BeginTabItem("FaceBone") then
		ImGui.Text("Hello")
		ImGui.EndTabItem()
		end
		if ImGui.BeginTabItem("FaceDecal") then
		ImGui.Text("Say")
		ImGui.EndTabItem()
		end
		if ImGui.BeginTabItem("FaceFile") then
		ImGui.Text("Babay")
		ImGui.EndTabItem()
		end

		ImGui.EndTabBar()
	end
	ImGui.End();
end

win:start()
```
![tabissueGIF](https://user-images.githubusercontent.com/1795930/86446007-9a5c7400-bd13-11ea-9f07-4cb69720b470.gif)

**Comment by YuDang1024** [comment]
> ```lua
> ImGui.Separator()
> ```

I am using Lua5.2, and now I have drawn a lot of components, and use them, only the current BeginTabBar has a problem, I will carefully check whether the exported interface is correct. I will update in time for any new discoveries

**Comment by sonoro1234** [comment]
> I am using Lua5.2

I meant the imgui-Lua library and version

**Comment by YuDang1024** [comment]
> > I am using Lua5.2
> 
> I meant the imgui-Lua library and version.
> Anyway, I guess it is related to the binding. So you should post the issue in the imgui-Lua repository.

I use toLua++ to export the C++ interface,
What is the address of the imgui-Lua repository？
Thank you！

**Comment by sonoro1234** [comment]
> I use toLua++ to export the C++ interface,

This is the information about the imgui-Lua library: you are generating your own imgui-Lua library with toLua++


> What is the address of the imgui-Lua repository？

In this case it would be your address!!!
Sorry, I cant help more with this as I dont know how toLua++ works. Perhaps it would be useful for you to know that there are already some Lua and LuaJIT libraries for Dear ImGui: https://github.com/ocornut/imgui/wiki/Bindings

**Comment by YuDang1024** [comment]
> > I use toLua++ to export the C++ interface,
> 
> This is the information about the imgui-Lua library: you are generating your own imgui-Lua library with toLua++
> 
> > What is the address of the imgui-Lua repository？
> 
> In this case it would be your address!!!
> 
> Sorry, I cant help more with this as I dont know how toLua++ works. Perhaps it would be useful for you to know that there are already some Lua and LuaJIT libraries for Dear ImGui: https://github.com/ocornut/imgui/wiki/Bindings

Thanks for your reply, I will continue to check my program, if I find the answer, I will come here to update

**Comment by ocornut** [comment]
You can google “imgui-lua” for existing bindings.
Closing this as this is probably a user or custom binding error. Feel free to keep posting here.

**Comment by YuDang1024** [comment]
I may have found some reasons:
I exported Imgui to Lua,
I gave the default values "NUll" and "0" for the second and third parameters of the BeginTabItem function,
This will cause him not to be rendered.
This is caused by an inappropriate default value set in my export interface
However, it seems that the flag is set incorrectly,
My TabBar has a close button on it,
I don’t want this button. I tried several combinations of flags, but it didn’t work. How should I set this up?
![image](https://user-images.githubusercontent.com/55690445/87112917-aca05a00-c29f-11ea-8991-72459cf84765.png)
